tests/test.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+set -exuo pipefail
+
+# This script runs disk image with qemu-system and run tmt against this vm.
+
+BOOTC_TEMPDIR="/tmp/tmp-bootc-build"
+SSH_OPTIONS=(-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ConnectTimeout=5)
+SSH_KEY=${BOOTC_TEMPDIR}/id_rsa
+
+ARCH=$(uname -m)
+case "$ARCH" in
+"aarch64")
+  qemu-system-aarch64 \
```

Comment:
Eventually this is going to need to use a shared library of some form. Can't we just point tmt at the disk image and let testcloud run it or so?